Modules/Core/Common/include/itkMacro.h

```diff
@@ -157,29 +140,39 @@ namespace itk
  * MSVC++ 6.0 _MSC_VER = 1200
  * MSVC++ 5.0 _MSC_VER = 1100
 */
-#if defined( _MSC_VER ) && ( _MSC_VER < 1310 )
-//#error "_MSC_VER < 1310 (MSVC++ 7.1) not supported under ITKv4"
+#if defined( _MSC_VER ) && ( _MSC_VER < 1600 )
+  #error "MSVC++ < 10.0 is not supported under ITKv5"
```

Comment:
`Visual Studio 2015 is the oldest version supported by ITKv5.`
VS versions are much better known then MSVC versions, so I think this message is friendlier.